src/lib/node/incoming_message.ts

```diff
@@ -94,6 +94,14 @@ export interface IncomingCommandNodeSetRawConfigParameterValue
   valueFormat?: ConfigValueFormat;
 }
 
+export interface IncomingCommandNodeGetRawConfigParameterValue
+  extends IncomingCommandNodeBase {
+  command: NodeCommand.getRawConfigParameterValue;
+  parameter: number;
+  bitMask?: number;
+  allowUnexpectedResponse?: boolean;
```

Comment:
`allowUnexpectedResponse` is only really needed during the interview, which the driver does internally. I'd probably not expose it to applications.